pkg/3scale/amp/operator/system_mysql_reconciler.go

```diff
@@ -25,7 +25,7 @@ func (r *SystemMySQLReconciler) Reconcile() (reconcile.Result, error) {
 	}
 
 	// DC
-	err = r.ReconcileDeploymentConfig(systemMySQL.DeploymentConfig(), reconcilers.CreateOnlyMutator)
```

Comment:
Using reconcilers.CreateOnlyMutator seemed originally wrong, as  SystemPostgreSQL Reconciler was using `DeploymentConfigResourcesMutator` instead of `reconcilers.CreateOnlyMutator`.

In new reconciler used in this case I've made it reconcile resources too

Reply:
:+1: